### ISSUE TITLE
Add Ul

### ISSUE THREAD

**Issue by davetapley** [fibo/trunx#64]
Can we get a `Ul` for `.menu-list`?

**Comment by fibo** [comment]
hi @davetapley 

good idea! Added `Ul`, published as v1.1.1 on npm.

Also added instructions about [how to use Trunx with Preact](https://github.com/fibo/trunx#use-trunx-with-preact).
I did an experiment on a side project and I was able to drastically reduce the bundle size with Preact. Of course mine is a particular use case, it is a very simple website and I am not using other dependencies ... but it works 💪 

Happy new Year!!

And let me know it something is wrong with the recent release.
